internal/host/validator.go

```diff
@@ -1110,41 +938,41 @@ func comparisonBuilder(value float64) string {
 	return equals
 }
 
-func (v *validator) printSufficientNetworkLatencyRequirementForRole(c *validationContext, status ValidationStatus) string {
+func (v *validator) hasSufficientPacketLossRequirementForRole(c *validationContext) (ValidationStatus, string) {
+
+	var status ValidationStatus
+	var message string
+
+	if c.infraEnv != nil {
+		status = ValidationSuccessSuppressOutput
```

Comment:
I think this function can be simplified. For example, you can return early here as this is the only place you get this status.